app/views/shared/_dev_mode.html.slim

```diff
@@ -0,0 +1,2 @@
+.bg-yellow.blue.center
```

Comment:
this can be simplified to a one-liner: `.bg-yellow.blue.h4.center 🚨 Development Mode Enabled 🚨`